docs/specs/config.yml

```diff
@@ -202,7 +202,7 @@ outputs:
 inputs:
   docfx.yml: |
     routes:
-      /: outputB
+      ./: outputB
```

Comment:
`/` is the root drive on linux (similar to `C:/` on windows), thus this needs to be `.` or `./`